client/state/jetpack-connect/reducer/test/jetpack-connect-site.js

```diff
@@ -2,11 +2,6 @@
 /**
  * External dependencies
  */
-import { expect } from 'chai';
-
-/**
- * Internal dependencies
```

Comment:
We probably should have kept the `Internal dependencies` comment here, and ditch the `External` one, correct?

Reply:
👍 thanks! c12bcb7b32aa85ea51cdb5179d4610cd7e9b030b